internal/app/fhir/r5/CodeSystem.go

```diff
@@ -39,9 +38,16 @@ func SerializeCodeSystemToFhir(cs *xo.CodeSystem) (*r5pb.CodeSystem, error) {
 		},
 	}
 
+	var definitionText string
+	if cs.Definitiontext.Valid {
+		definitionText = cs.Definitiontext.String
+	} else {
+		definitionText = ""
+	}
```

Comment:
```suggestion
	definitionText := ""
	if cs.Definitiontext.Valid {
		definitionText = cs.Definitiontext.String
	}
```
Any idea if the concise version is considered to be more idiomatic in Go? Feel free to disregard if the verbosity is preferable!

Reply:
Yes, much better 👍🏻